src/board.js

```diff
@@ -5,31 +5,39 @@ export default class Board extends React.Component {
     renderSquare(i) {
         return (
             <Square 
+                key={i}
                 value={this.props.squares[i]}
                 onClick={() => this.props.onClick(i)}
             />
         );
     }
 
-    render() {
+    side() {
```

Comment:
Instead of calling the side function 2x per, you could call it once per component render

componentDidMount(){
this.setState({side: Math.sqrt(this.props.squares.length);
}

and then you could refer to it as this.state.side